bin/get-google-analytics-pageviews

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env ruby
+require "csv"
+require "google/apis/analyticsreporting_v4"
+require "googleauth"
+require "yaml"
+
+GA_REPORTING = Google::Apis::AnalyticsreportingV4
+
+def load_config_data
+  YAML.load(File.open("config/ga_env.yml")).each do |key, value|
+    ENV[key.to_s] = value
+  end
+end
+
+def matrix_file
```

Comment:
I wonder if it'll be easier/nicer if we can pass the input filename and the output filename as an argument (`ARGV`) to this script? In `generate-matrix.sh` we could then pass them as the same value and save having to do the `mv` or have a temporary `new_matrix.csv` file.